src/app/components/input/input.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, EventEmitter, Input, Output, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-input',
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.scss']
+})
+export class InputComponent {
+
+  @Output() valueChange = new EventEmitter<string>;
+  @Input() placeHolder: string = '';
+  @Input() inputType: string = '';
+  @Input() fieldsetLabel:string = '';
+  @Input() iconName = '';
+  @Input() inputWidth:string = '';
+  @Input() inputBorderColor:string = '#2C85D8';
+  
+  value:string = '';
+  showPassword:boolean = false;
```

Comment:
é bom usar public ou private antes de uma variável do componente dependendo da intenção
https://stackoverflow.com/questions/37506343/private-and-public-in-angular-component

Reply:
ok. corrigido